features/step_definitions/breadcrumbs_steps.rb

```diff
@@ -0,0 +1,14 @@
+Given(/^the page exists$/) do
```

Comment:
Given(/^there exists (something here) content item$/) do

Reply:
Or instead of doing all of that you can simply reuse an existing step, please, see a comment above.